app/route_rangers_api/urls.py

```diff
@@ -1,11 +1,15 @@
 from django.urls import path
-
 from . import views
-from route_rangers_api.views import MarkersMapView
+from views import MarkersMapView
```

Comment:
probably delete this because we are no longer using this view if I understood correctly

Reply:
fixed